air/src/graph/mod.rs

```diff
@@ -23,6 +23,18 @@ impl core::ops::Add<usize> for &NodeIndex {
     }
 }
 
+impl From<usize> for NodeIndex {
```

Comment:
There is a blanket implementation `Into<U> for T` when we implement `From<T> for U` [source](https://doc.rust-lang.org/std/convert/trait.From.html#generic-implementations). This mean you can write
```rust
let n = NodeIndex::default();
let u: usize = n.into().
```